packages/xchain-ethereum/src/client.ts

```diff
@@ -118,36 +124,29 @@ export default class Client implements XChainClient, EthereumClient {
     ethplorerUrl = 'https://api.ethplorer.io',
     ethplorerApiKey = 'freekey',
     explorerUrl,
-    phrase,
+    phrase = '',
+    rootDerivationPaths = {
+      mainnet: `m/44'/60'/0'/0/`,
+      testnet: `m/44'/60'/0'/0/`, // this is INCORRECT but makes the unit tests pass
```

Comment:
> // this is INCORRECT but makes the unit tests pass

Could you clarify it ^?

Reply:
i could be wrogn about this one...but i thought the general consensus is that a given seed phrase should generate different addresses on testnet and  mainnet 
https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#account

the previous implementation  was using the same derivation path  for testnet and mainnet, but maybe this normal for ethereum? i just noticed metamask uses the same address  on testnet and mainnet...

i'm thinkign i should remove this comment?